src/packages/ganache/api-extractor.json

```diff
@@ -69,15 +69,12 @@
     "@ganache/ethereum-transaction",
     "@ganache/ethereum-block",
     "@ganache/console.log",
-    "@ethereumjs/vm",
     "@ethereumjs/statemanager",
-    "@ethereumjs/evm",
     "@ethereumjs/util",
     "@ethereumjs/tx",
     "@ethereumjs/common",
     "@ethereumjs/block",
     "@ethereumjs/blockchain",
-    "@ethereumjs/trie",
     "@ethereumjs/ethash",
```

Comment:
I don't understand why _some_ @ethereumjs packages get treated as "bundledPackages"?

Reply:
Because those packages don't break our types ¯\_(ツ)_/¯

API extractor doesn't work correctly for the vm, evm, and trie packages, but it works fine for the rest. We'd rather have them all here in the extractor config than in the package.json as a dependency, because adding to the package.json increases our total bundle size. But alas, since API extractor doesn't handle these packages correctly, we're removing them.